kubernetes/loculus/templates/_possiblePriorityClassName.tpl

```diff
@@ -0,0 +1,5 @@
+{{- define "possiblePriorityClassName" -}}
+{{- if .Values.priorityClassName }}
+priorityClassName: {{ .Values.priorityClassName }}
```

Comment:
Maybe make the value a little bit more self explanatory, e.g. `.Values.podPriorityClassName`

otherwise LGTM